services/frontend-v3/src/components/admin/dashboardGraphs/DashboardGraphsView.jsx

```diff
@@ -1,10 +1,19 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Card, Row, Col } from "antd";
-import { Chart, Geom, Axis, Tooltip, Coord, Legend } from "bizcharts";
 import { FormattedMessage, injectIntl } from "react-intl";
 import { IntlPropType } from "../../../utils/customPropTypes";
 
+const { Chart, Geom, Axis, Tooltip, Coord, Legend, registerTheme } =
+  typeof document === "undefined" ? {} : require("bizcharts");
+
+if (registerTheme) {
+  //   registerTheme("default", {
+  //     defaultColor: "#505050",
```

Comment:
Is this commented code needed ?

Reply:
Oups, I'll remove it - It was to change the colors for the charts